### PR TITLE
added a extract that writes legacy vtk files

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

**Comment by cyrush** [comment]
@nicolemarsaglia took a pass at creating a new extract that writes legacy vtk files, one per domain, that are grouped by a `.visit` file.

Can you give it a once over?

File: src/tests/ascent/t_ascent_mpi_vtk_file_extract.cpp [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+using namespace std;
+using namespace conduit;
+using namespace ascent;
+
+
+index_t EXAMPLE_MESH_SIDE_DIM = 10;
+
+//-----------------------------------------------------------------------------
+TEST(ascent_vtk_file_extract, basic_mulit_domain)
```
Comment (nicolemarsaglia) [inline src/tests/ascent/t_ascent_mpi_vtk_file_extract.cpp:37]: The mulit domain -- structured in the front, unstructured in the back ;)

**Review by nicolemarsaglia — COMMENTED** [review]
Things looked good! Way to get this put together so quickly!

File: src/tests/ascent/t_ascent_mpi_vtk_file_extract.cpp (hunk 1 above)
Comment (cyrush) [reply src/tests/ascent/t_ascent_mpi_vtk_file_extract.cpp:37]: now I have to keep the typo :-)

**Pull request merged**